python/lsst/afw/image/exposure/exposureContinued.py

```diff
@@ -99,14 +99,13 @@ def setVariance(self, variance):
 
     writeFitsWithOptions = exposureWriteFitsWithOptions
 
-    # TODO DM-18544: activate this once `deprecated` becomes available
-    # @deprecated(reason="Replaced with getPhotoCalib (will be removed in 18.0)", category=FutureWarning)
-    # def getCalib(self, *args, **kwargs):
-    #     return self._getCalib(*args, **kwargs)
+    @deprecated(reason="Replaced with getPhotoCalib (will be removed after v18)", category=FutureWarning)
+    def getCalib(self, *args, **kwargs):
+        return self._getCalib(*args, **kwargs)
 
-    # @deprecated(reason="Replaced with setPhotoCalib (will be removed in 18.0)", category=FutureWarning)
-    # def setCalib(self, *args, **kwargs):
-    #     return self._setCalib(*args, **kwargs)
+    @deprecated(reason="Replaced with setPhotoCalib (will be removed after v18)", category=FutureWarning)
+    def setCalib(self, *args, **kwargs):
+        self._setCalib(*args, **kwargs)
```

Comment:
Why force the wrapped methods to be all underscores instead of just following the style guide

    setCalib = deprecated(reason="Replaced with setPhotoCalib (will be removed after v18)", category=FutureWarning)(setCalib)

You had to make a lot of changes in the C++ file...

Reply:
Because that doesn't work for methods on classes due to a docstring overwriting problem. @ktlim has what he thinks is a fix for upstream `deprecate`, but getting that into the stack is a problem because of upgrading the conda env being complicated.